### PR TITLE
Fail tests when adding new props to fpti

### PR DESCRIPTION
**Pull request by zacharyhill**
DTSHOPSDK-610

**Purpose of this PR:**
We wanted to ensure our docs stay updated when new FPTI properties are added to events. In order to do this, we are checking included properties and supplying to link to update docs when the tests fail.

**What it will look like when any of these tests fail:**
image

### CODE REVIEW

File: src/lib/shopping-fpti/event-handlers.js [hunk 1]
```diff
@@ -8,16 +8,41 @@ function findConfigurationAttribute(config : Config, payload : Object = {}, attr
   return shopperConfig[attribName] || payload[attribName];
 }
 
+export const includedAttributes = [
```
Comment (ukalbhor) [inline src/lib/shopping-fpti/event-handlers.js:11]: minor : maybe `allowedAttributes` ?

File: src/lib/shopping-fpti/event-handlers.js [hunk 2]
```diff
@@ -8,16 +8,41 @@ function findConfigurationAttribute(config : Config, payload : Object = {}, attr
...
+  'product_price',
+  'product_brand',
+  'product_category_name',
+  'product_category_id',
+  'product_discount',
+  // purchase
+  'amount',
+  // set properties
+  'currency',
+];
```
Comment (ukalbhor) [inline src/lib/shopping-fpti/event-handlers.js:36]: This is totally fine .... but this seems like the perfect fit for a `@flow` type where we get this check for 'free'.

Comment (zacharyhill) [reply src/lib/shopping-fpti/event-handlers.js:36]: Can we use the flow type and not affect the functionality of the code? Is flow types something we can only apply to tests?

File: src/lib/shopping-fpti/event-handlers.js (hunk 1 above)
Comment (zacharyhill) [reply src/lib/shopping-fpti/event-handlers.js:11]: Yeah, that makes more sense. I'll update it.

File: src/lib/shopping-fpti/event-handlers.js (hunk 2 above)
Comment (ukalbhor) [reply src/lib/shopping-fpti/event-handlers.js:36]: hmm yeah ..... the tests need to fail which will be complicated with a `@flow` type

**Review by ukalbhor — APPROVED** [review]

**Pull request merged**